presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -647,6 +658,7 @@ INTO: 'INTO';
 IO: 'IO';
 IS: 'IS';
 ISOLATION: 'ISOLATION';
+JAVA: 'JAVA';
```

Comment:
It's a reserved word now?

Reply:
These are non-reserved words. They are in the nonReserved list. I actually don't know what's the purpose of this list here. I was guessing that this was just defining the string representation of keywords appeared in the grammar. But I didn't find any reference on what they actually are. 😳

Reply:
This is a lexer rule that defines text 'JAVA' in query string as a token named `JAVA`. This does not specify whether it is a reserved words or not.